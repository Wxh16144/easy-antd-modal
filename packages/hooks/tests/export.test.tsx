describe('export', () => {
  it('should work', async () => {
    const all = await import('@wuxh/use-modal-enhanced');

    expect(Object.keys(all)).toMatchSnapshot();
  });
});
