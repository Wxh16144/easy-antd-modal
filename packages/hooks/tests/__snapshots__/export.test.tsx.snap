// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`export > should work 1`] = `
[
  "useBoolean",
  "useLatestFunc",
  "useModalEnhanced",
  "EasyAntdModalContext",
  "useEasyAntdModal",
  "EasyAntdModalProvider",
  "has",
  "omit",
  "isElement",
  "isDOMTypeElement",
]
`;
